app/models/comment.rb

```diff
@@ -1,7 +1,8 @@
 class Comment < ApplicationRecord
-  validates :content, presence: true, length: { maximum: 200,
-                                                too_long: '200 characters in comment is the maximum allowed.' }
+  validates :content,
+            presence: true,
+            length: { maximum: 200, too_long: '200 characters in comment is the maximum allowed.' }
```

Comment:
The validation test for this is not complete. You have omitted tests for length. Hint: You can use [shoulda Matchers](http://matchers.shoulda.io/docs/v4.0.1/Shoulda/Matchers/ActiveModel.html) for this.
Make a similar change for `post.rb` and `user.rb` files.